apps/judicial-system/web/src/routes/Shared/SignedVerdictOverview/SignedVerdictOverviewForm.tsx

```diff
@@ -131,60 +134,9 @@ const SignedVerdictOverviewForm: React.FC<Props> = (props) => {
       : 'Gæsluvarðhald virkt'
   }
 
-  const subtitleForCase = (theCase: Case) => {
-    const isTravelBan =
-      theCase.decision === CaseDecision.ACCEPTING_ALTERNATIVE_TRAVEL_BAN ||
-      theCase.type === CaseType.TRAVEL_BAN
-
-    if (
-      theCase.decision === CaseDecision.REJECTING ||
-      theCase.decision === CaseDecision.DISMISSING ||
-      isInvestigationCase(theCase.type)
-    ) {
-      return `Úrskurðað ${formatDate(
-        theCase.courtEndTime,
-        'PPP',
-      )} kl. ${formatDate(theCase.courtEndTime, TIME_FORMAT)}`
-    }
-
-    if (theCase.isValidToDateInThePast) {
-      return (
-        <>
-          <Box component="span" display="block">
-            {`Úrskurðað ${formatDate(
-              theCase.rulingDate,
-              'PPP',
-            )} kl. ${formatDate(theCase.rulingDate, TIME_FORMAT)}`}
-          </Box>
-          <Box component="span">
-            {`${
-              isTravelBan ? 'Farbann' : 'Gæsla' // ACCEPTING
-            } rann út ${formatDate(
-              theCase.validToDate,
-              'PPP',
-            )} kl. ${formatDate(theCase.validToDate, TIME_FORMAT)}`}
-          </Box>
-        </>
-      )
-    }
-
+  const canModifyCaseDates = () => {
     return (
-      <>
-        <Box component="span" display="block">
-          {`Úrskurðað ${formatDate(theCase.rulingDate, 'PPP')} kl. ${formatDate(
-            theCase.rulingDate,
-            TIME_FORMAT,
-          )}`}
-        </Box>
-        <Box component="span">
-          {`${
-            isTravelBan ? 'Farbann' : 'Gæsla' // ACCEPTING
-          } til ${formatDate(theCase.validToDate, 'PPP')} kl. ${formatDate(
-            theCase.validToDate,
-            TIME_FORMAT,
-          )}`}
-        </Box>
-      </>
+      user?.role === UserRole.JUDGE && workingCase.type === CaseType.CUSTODY
```

Comment:
Consider who should be allowed to change.

- Here, only judges are allowed to change dates.
- Server side, both judges and registrar are allowed to change dates.
- In the examples from Guðný in Asana prosecutors are changing dates.

Who should actually be allowed to change dates?

Reply:
Note also that in Guðný's example, the prosecutor's office is also shown.